AssetRipper.Library/Configuration/SpriteExportMode.cs

```diff
@@ -3,7 +3,13 @@
 	public enum SpriteExportMode
 	{
 		/// <summary>
-		/// Export in the native asset format
+		/// Export as yaml assets which can be viewed in the editor.
+		/// This is the only mode that can ensure a precise recovery of metadata of all sprites.
+		/// <see href="https://github.com/trouger/AssetRipper/issues/2"/>
+		/// </summary>
+		Yaml,
```

Comment:
Merge Yaml and Native

Reply:
If I change the behavior of Native to my implementation of Yaml sprite assets, then the original "export sprites into texture import settings" code should be deleted. Is that ok?

Reply:
On second thought, leave it as three. I can see three valid export modes here:
* Yaml, where a sprites are exported independently of textures as yaml assets.
* Native, where sprites are folded into texture import settings.
* Texture2D, where sprites cut up a texture and get converted into individual textures.